src/jsonLogic.js

```diff
@@ -212,3 +234,71 @@ function handleNestedObjectForComputedValues(values, formValues, parentID, valid
     })
   );
 }
+
+function buildSampleEmptyObject(schema = {}) {
+  const sample = {};
+  if (typeof schema !== 'object' || !schema.properties) {
+    return schema;
+  }
+
+  for (const key in schema.properties) {
+    if (schema.properties[key].type === 'object') {
+      sample[key] = buildSampleEmptyObject(schema.properties[key]);
+    } else if (schema.properties[key].type === 'array') {
+      const itemSchema = schema.properties[key].items;
+      sample[key] = buildSampleEmptyObject(itemSchema);
+    } else {
+      sample[key] = true;
+    }
+  }
+
+  return sample;
+}
+
+function validateInlineRules(jsonSchema, sampleEmptyObject) {
+  const properties = (jsonSchema?.properties || jsonSchema?.items?.properties) ?? {};
+  Object.entries(properties)
+    .filter(([, property]) => property['x-jsf-logic-computedAttrs'] !== undefined)
+    .forEach(([fieldName, property]) => {
+      Object.entries(property['x-jsf-logic-computedAttrs'])
+        .filter(([, value]) => typeof value === 'object')
+        .forEach(([key, item]) => {
+          Object.values(item).forEach((rule) => {
+            checkRuleIntegrity(
+              rule,
+              fieldName,
+              sampleEmptyObject,
+              (item) =>
+                `"${item.var}" in inline rule in property "${fieldName}.x-jsf-logic-computedAttrs.${key}" does not exist as a JSON schema property.`
+            );
+          });
+        });
+    });
+}
+
+function checkRuleIntegrity(
```

Comment:
todo: A jsDoc with explanation + simple example across all these new functions would be very useful

Reply:
lots of small tweaks in f3b0895ec90890913004d538d9c2d3fb4e528ba0 for this, LMK!